app/src/main/java/com/onboarding/parkingsystemjava/database/ParkingDatabase.java

```diff
@@ -41,4 +44,31 @@ public int getParkingLots() {
     public void setParkingLots(int parkingLots) {
         this.parkingLots = parkingLots;
     }
+
+    public int updateDatabase() {
+        Calendar currentTime = new GregorianCalendar();
+        HashMap<Integer, ArrayList<Reservation>> reservations = this.getReservations();
+        int deleted = ConstantUtils.INT_ZERO;
+        for (int i = ConstantUtils.INT_ZERO; i <= this.parkingLots; i++) {
+            if (reservations.containsKey(i)) {
+                deleted += searchLotForOldReservation(currentTime, this.reservations.get(i));
```

Comment:
why you are adding "+=" for this variable? you could only assign the value

Reply:
because I need to accumulate the amount of deletions on each parking lot (key). If I assign the value, the next iteration will override the previous deleted value.